src/wfutil.c

```diff
@@ -1617,3 +1619,78 @@ BOOL TypeAheadString(WCHAR ch, LPWSTR szT)
 
    return ich != 0;
 }
+
+// Search the list of slots if path would be parent of already existing drive == circularity
+BOOL FindUNCLoop(LPCTSTR path)
+{
+   for (DWORD dwDriveIndex = OFFSET_UNC; dwDriveIndex < MAX_DRIVES; ++dwDriveIndex) {

```

Comment:
Sorry no. We are in 2023 and applying a 34 year old coding style seems odd to me.

It makes things unreadable. One day we have to face the present ;-)

I know you need the old compiler for you W2K & WXP branch. Anyhow the whole Winfile has lots of C89 violations and also contains wfgoto.cpp which you have to exclude from the makefile to get it compiling. 

So you have to anyhow change something, when you bring it to your branch. To ease your life you could start with https://github.com/schinagl/winfile/tree/malxau_hardlink_junctions which I took from you and made the whole hardlink/sysmlink stuff compile with C89.

Reply:
Stepping back a bit, my motivation is to maximize the potential contributors and potential users of the project.  Reasonable people can disagree how to best achieve those goals, but those goals underpin everything I'm doing in this project.

When I made the previous comment, I believed that #165 had removed C99 variable declarations and replaced them with C89, and C99 was being gradually reintroduced.  After reviewing this and related PRs, it appears they never merged, which takes care of the core of my objection.  Note these did not remove wfgoto; they attempted to use earlier C++ standards to support compiling on XP.

Reading the comments in the PRs though stings a bit, because it shows potential contributors being discouraged and openly giving up on the project.  It's not surprising that a project like this would attract retro enthusiasts, and keeping a large group of contributors implies having people with different goals working together.  I hope that you and I share that perspective: if we didn't, it would be easy enough to archive this repo, leaving it read only, and let anyone maintain their own fork with no collaboration.  Working together means accepting that not everyone wants the same thing, but if we have enough overlap, we can still improve things for each other.

Reply:
So you mean that people do not develop on this repo because it is not in C89 ??? This is hilarious.

I am totally confused reading you above writing.
fyi: @craigwi